QRScanner/QRScannerView.swift

```diff
@@ -86,6 +100,7 @@ public class QRScannerView: UIView {
                 self?.session.stopRunning()
             }
         }
+        torchActiveObservation = nil
```

Comment:
BTW, `.invalidate()` is dangerous so I fixed.
`NSKeyValueObservation` should be nil at `deinit`.
And now array is unnecessary so I changed the property Array to Optional.
https://stackoverflow.com/questions/52479027/nskeyvalueobservation-cannot-remove-an-observer-for-the-key-path-from-object-be